src/main/java/de/bonndan/nivio/input/Seed.java

```diff
@@ -13,30 +13,27 @@
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
+import java.util.Optional;
 
 /**
  * Evaluation of the SEED environment variable.
  */
 public class Seed {
 
-    public static final String DEMO = "DEMO";
     private static final Logger logger = LoggerFactory.getLogger(Seed.class);
-    @Value("${SEED:}")
-    private String seed;
 
-    public Seed() {
-    }
+    private final String seed;
 
-    public Seed(String seed) {
-        this.seed = seed;
+    public Seed(Optional<String> seed) {
+        this.seed = seed.orElse(null);
```

Comment:
This `Optional` is the return value of the `ConfigurableEnvVars` enums method `value()`:
````
public Optional<String> value() {
   return Optional.ofNullable(System.getenv(name()));
}
````
If this is transformed to `null` anyways, why bother using the `Optional`? I think this only makes sense, if the `Optional`is really used to handle some kind of state.

Reply:
Okay, I saw that in e.g. GitLabHandler, you are really using isPresent checks.